rooms.js

```diff
@@ -500,8 +499,7 @@ class GlobalRoom {
 	}
 	getRooms(user) {
 		let roomsData = {official:[], pspl:[], chat:[], userCount: this.userCount, battleCount: this.battleCount};
-		for (let i = 0; i < this.chatRooms.length; i++) {
-			let room = this.chatRooms[i];
+		for (const room of this.chatrooms) {
```

Comment:
Did you test this? This is crashing whenever a user tries to get the roomlist
```
TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined
at GlobalRoom.getRooms (/home/ubuntu/workspace/rooms.js:502:26)
```

Reply:
Honestly, this is why we need TypeScript

Reply:
😦